README.md

```diff
@@ -77,13 +77,14 @@ The following CMake configuration options are supported.
 | CMAKE_BUILD_TYPE     | Release        |            | With full optimizations.                   |
 |                      | Debug          |            | With debug symbols.                        |
 |                      | RelWithDebInfo |            | With full optimizations and debug symbols. |
-| CMAKE_INSTALL_PREFIX | <string>       | /usr/local | The directory to install the library in.   |
+|                      | Dev/DevDebug   |            |                                            |
+| CMAKE_INSTALL_PREFIX |                | /usr/local | The directory to install the library in.   |
 | USE_TPM              | ON, OFF        | ON         | Build with support for using a TPM 2.0     |
 | BUILD_TOOL           | ON, OFF        | ON         | Build tool.                                |
 | BUILD_SHARED_LIBS    | ON, OFF        | ON         | Build shared libraries.                    |
 | BUILD_STATIC_LIBS    | ON, OFF        | OFF        | Build static libraries.                    |
 | BUILD_TESTING        | ON, OFF        | ON         | Build the test suite.                      |
-| STATIC_SUFFIX        | <string>       | <none>     | Appends a suffix to the static lib name.   |
+| STATIC_SUFFIX        |                |            | Appends a suffix to the static lib name.   |
```

Comment:
Why did these go away?

Reply:
These two are related. While trying to make separate rows, GitHub's preview was showing an incorrect final table, so I was trying to fix something that wasn't broken, this was a part of it.